### ISSUE TITLE
The GitHub icon link in the navigation bar does not have a text label for screen readers

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/videojs.com#69]
This line:

https://github.com/videojs/videojs.com/blob/master/_harp/_nav.ejs#L19

inserts the GitHub icon on the right side of the navigation bar, with a link to https://github.com/videojs/video.js. But the link is unlabeled for screen reader users, so, for example, JAWS announces it as "videojs/video.js" (based on the link).

Recommend that something like `aria-label="Video.js on GitHub"` be added to the link element.

**Comment by heff** [comment]
@OwenEdwards made a quick PR for this. Give it a 👍 and I'll merge it in.

FWIW, I didn't even open my code editor to make this change. I just hit the edit button on the file view of _nav.ejs you linked to, and told it to make a new PR. Try that out the next small change you have. Gonna go make sure you have maintainer access to this repo.

**Comment by OwenEdwards** [comment]
Thanks @heff, I'll do that. I was in the middle of tracking down another issue and just wanted to make sure I'd noted this issue before I forgot it. Also, I wasn't clear whether making an edit through the GitHub interface would make a new PR or just change apply the change without anyone else's approval, so it's good to know that that's possible.

**Comment by heff** [comment]
Nice! Didn't mean for that to come off as critical if it did. :) That was the first PR I've made through github and thought it was cool.

**Comment by gkatsev** [comment]
GitHub allows you to make it a direct commit or make it a PR, so, yeah, for simple changes you often can just do it in github :D

Also, I'm going to be looking into switching to netlify, so, we could then even deploy from the UI because we could then make a PR from master to a deploy/gh-pages branch and have netlify publish it.

**Comment by heff** [comment]
Oooh, fancy!
